test/common/config/http_subscription_test_harness.h

```diff
@@ -70,12 +70,11 @@ class HttpSubscriptionTestHarness : public SubscriptionTestHarness {
                                                         Http::AsyncClient::Callbacks& callbacks,
                                                         const Http::AsyncClient::RequestOptions&) {
           http_callbacks_ = &callbacks;
-          EXPECT_EQ("POST", std::string(request->headers().Method()->value().getStringView()));
+          EXPECT_EQ("POST", std::string(request->headers().getMethodValue()));
```

Comment:
I noticed on your other new accessor PR that these unnecessary std::strings could be moved to a beginner clean-up task. This is just test code, so it's nbd, but I'd add this as clean-up in the issue (maybe just reference this whole PR)

Reply:
good call: https://github.com/envoyproxy/envoy/issues/11318

Reply:
@asraa  I am not able to get where is getMethodValue() defined. Can you please help out.